tern/classes/image_layer.py

```diff
@@ -106,6 +112,13 @@ def os_guess(self):
     def os_guess(self, os_guess):
         self.__os_guess = os_guess
 
+    @files_analyzed.setter
+    def files_analyzed(self, x):
+        if bool(x):
+            self.__files_analyzed = x
+        else:
+            raise ValueError('files_analyzed should be boolean')
+    
```

Comment:
Could you move the setter with the `@property` so it is easier to find when looking at this code later?